velox/exec/benchmarks/VectorHasherBenchmark.cpp

```diff
@@ -177,6 +177,32 @@ BENCHMARK_RELATIVE(computeValueIdsFlatStrings) {
   benchmarkComputeValueIdsForStrings(true);
 }
 
+BENCHMARK(computeValueIdsLowCardinalityLargeBatchSize) {
+  folly::BenchmarkSuspender suspender;
+
+  vector_size_t cardinality = 300;
+  vector_size_t batchSize = 30'000'000;
+  BenchmarkBase base;
+
+  std::vector<std::optional<int64_t>> data(batchSize);
+  for (int i = 0; i < batchSize; i++) {
+    data[i] = i % cardinality;
+  }
+  auto values = base.vectorMaker().dictionaryVector<int64_t>(data);
```

Comment:
Would you add another benchmark for the case where not all base values are used? Let's say out of 300 values only 299 are used. Will there be any difference in performance?

It would be nice to update PR description to include benchmark results before and after the changes.

Reply:
- Added another benchmark.
- Please check update PR desc with Benchmark outcome.